src/main/java/de/presti/ree6/commands/CommandEvent.java

```diff
@@ -134,6 +135,16 @@ public void reply(MessageCreateData message, int deleteSecond) {
         Main.getInstance().getCommandManager().sendMessage(message, deleteSecond, getChannel(), getInteractionHook());
     }
 
+    /**
+     * Get a Message from the Guild specific Language Setting.
+     * @param key the Key of the Message.
+     * @param parameters the Parameters to replace in the Message.
+     * @return the Message.
+     */
+    public String getResource(String key, Object... parameters) {
```

Comment:
This and all methods in LanguageService should be renamed because it collides with the method that every class inherits from the object class.